android-agent/src/main/java/io/opentelemetry/android/config/OtelRumConfig.java

```diff
@@ -179,14 +194,33 @@ public OtelRumConfig setSlowRenderingDetectionPollInterval(Duration duration) {
         return this;
     }
 
-    /** Returns true if crash reporting is enabled. */
+    /**
+     * Returns true if crash reporting is enabled.
+     */
     public boolean isCrashReportingEnabled() {
         return crashReportingEnabled;
     }
 
-    /** Call this method to disable crash reporting. */
+    /**
+     * Call this method to disable crash reporting.
+     */
     public OtelRumConfig disableCrashReporting() {
         crashReportingEnabled = false;
         return this;
     }
+
+    /**
+     * Call this method to set session timeout in minutes
+     */
+    public OtelRumConfig setSessionTimeoutMinutes(int sessionTimeoutMinutes) {
+        this.sessionTimeoutMinutes = sessionTimeoutMinutes;
+        return this;
+    }
+
+    /**
+     * Call this method to retrieve session timeout
+     */
+    public int getSessionTimeoutMinutes() {
+        return sessionTimeoutMinutes;
+    }
```

Comment:
Prefer to use the general purpose `Duration` instead of explicit minutes.

Reply:
➕

Reply:
Done fixed ✅